app/javascript/components/public_health/PatientsTable.js

```diff
@@ -154,6 +175,8 @@ class PatientsTable extends React.Component {
     if (await confirmDialog('Are you sure you want to clear all filters? All active filters and searches will be cleared.')) {
       localStorage.removeItem(`SaraFilter`);
       localStorage.removeItem(`SaraPage`);
+      localStorage.removeItem(`SaraSortField`);
```

Comment:
Why would these be cleared on filter clear?

Reply:
For this particular case actually, I'm wondering if it makes sense to reset the sort here. With the current intention of this PR, the user would only be able to rest the sort back to it's original ordering (based on Sara Alert ID) if a user either (1) goes to a different tab or (2) switches workflows. If we include a reset of the sort here, that would allow for the user to revert back to the original ordering without having to switch tabs or workflows (since there is no sortable column for Sara Alert ID).

If you think this could still be open for discussion, perhaps I should ask the PO team about this? Feel free to let me know your thoughts.

Reply:
Per feedback from the PO team, the Clear Filters button is expected to also reset the sort setting. So these lines will be included.